components/coachindicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@spectrum-css/coachindicator",
+  "version": "1.0.0",
+  "description": "The Spectrum CSS coachindicator component",
+  "license": "Apache-2.0",
+  "author": "Adobe",
+  "homepage": "https://opensource.adobe.com/spectrum-css/coachindicator",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git",
+    "directory": "components/coachindicator"
+  },
+  "bugs": {
+    "url": "https://github.com/adobe/spectrum-css/issues"
+  },
+  "main": "dist/index-vars.css",
+  "scripts": {
+    "build": "gulp"
+  },
+  "peerDependencies": {
+    "@spectrum-css/button": ">=10",
+    "@spectrum-css/buttongroup": ">=5 <=6",
```

Comment:
Is there a reason why this is `<=6`?

Reply:
These actually aren't even needed for coach indicator, I've removed them.